### PR TITLE
Ingredients : increment/decrement de quantities

### PR DESCRIPTION
**Pull request by DuboisCecile**
Still have a problem with line 323 of CustomizedFoodPage.js :
With {ingr.quantity} the quantity stays to 0, despite the fact that the array of ingredients is updated
When I try to use a state, it changes all the quantities (of all ingredients) each time I add 1 portion.

(Decrement is not working either, but it is probably because the quantity is stuck to 0, and it shouldn't decrement when the quantity is 0. So it basically does nothing...)

### CODE REVIEW

**Comment by DuboisCecile** [comment]
Problem fixed !

We can now add and remove ingredients : the quantities are updated and the image layer of each ingredient is added on the pizza, or removed when quantity of this ingredient = 0

File: src/components/PizzaChange.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+export default function PizzaChange(props) {
+  const ChosenIngredientsList = Object.keys(props).map((i) => props[i]);
+  return (
+    <div>
+      <div className="pizza-mockup-container">
+        {ChosenIngredientsList.map((ingr) =>
```
Comment (ComicScrip) [inline src/components/PizzaChange.js]: camelCase, please :)
PascalCase should be used for react components only

File: src/screens/CustomizedFoodPage.js [hunk 2]
```diff
@@ -156,45 +205,133 @@ const populateingredients = (array) => {
...
+            id: ingredToAdd[0].id,
+            name: ingredToAdd[0].name,
+            imgsrc: ingredToAdd[0].imglayer,
+            quantity: 1,
+            serving: ingredToAdd[0].serving,
+            kcal100: ingredToAdd[0].kcal100,
+          },
+        ]);
+      } else {
+        ingredToUpdate[0].quantity += 1;
```
Comment (ComicScrip) [inline src/screens/CustomizedFoodPage.js]: mutations are forbidden on react state data

Comment (ComicScrip) [reply src/screens/CustomizedFoodPage.js]: prefer mapping over the array in the state to create a new version with the quantity updated

**Review by ComicScrip — CHANGES_REQUESTED** [review]

File: src/screens/CustomizedFoodPage.js (hunk 2 above)
Comment (ComicScrip) [reply src/screens/CustomizedFoodPage.js]: or use https://immerjs.github.io/immer/#quick-example

**Comment by ComicScrip** [comment]
@DuboisCecile can I merge this or do you still have bugs ?

**Comment by DuboisCecile** [comment]
Yes, you can merge it, (yesterday's bugs were on another branch, for a future PR)
Thanks !

**Pull request merged**